### PR TITLE
Fix tensorrt-test

### PR DESCRIPTION
**Pull request by DVEfremov**

### CODE REVIEW

File: program/tensorrt-test/tensorrt-test.cpp [hunk 1]
```diff
@@ -133,15 +133,17 @@ int main( int argc, char** argv )
...
+    printf("\n[tensorrt-test] Start imageNet::Create...");
     // Create an imageNet object.
     imageNet* net = imageNet::Create(
                         caffe_model_val,
                         caffe_weights_val,
                         imagenet_mean_bin_val,
                         imagenet_synset_words_txt_val,
                         "data", "prob",
-                        !tensorrt_enable_fp16
+                        1//!tensorrt_enable_fp16
```
Comment (psyhtest) [inline program/tensorrt-test/tensorrt-test.cpp]: Wait, why always 1 despite the flag?

File: program/tensorrt-test/tensorrt-test.cpp [hunk 2]
```diff
@@ -167,6 +169,8 @@ int main( int argc, char** argv )
             const char* sample_imagenet_val_file = "ILSVRC2012_val_00002212.JPEG"; // 00002212 with AlexNet: top1="no", top5="yes"
             char* imagenet_val_path = (char*) malloc(strlen(imagenet_val_dir_val) + strlen(sample_imagenet_val_file) + 2);
             size_t num_images = 0;
+
+            printf("\n[tensorrt-test] Start scaning dire: %s\n", imagenet_val_path);
```
Comment (psyhtest) [inline program/tensorrt-test/tensorrt-test.cpp]:
```
printf("\n[tensorrt-test] Scanning directory \'%s\'\n", imagenet_val_path);
```

Comment (DVEfremov) [reply program/tensorrt-test/tensorrt-test.cpp]: ok

File: program/tensorrt-test/tensorrt-test.cpp (hunk 1 above)
Comment (DVEfremov) [reply program/tensorrt-test/tensorrt-test.cpp]: it's batchsize - will add comment

**Comment by DVEfremov** [comment]
ok I've fixed comments as we discussed I'm merging ...

**Pull request merged**